packages/template-blank-svelte/.npmignore

```diff
@@ -0,0 +1,20 @@
+npm-debug.log
+.DS_Store
+
+*.js.map
+*.js
```

Comment:
It might be safer to remove the *.js here. I have been bitten before by this excluding other config files

Reply:
👍  - would ideally refactor all to use `files` in `package.json` rather than `npmignore`